teams/infra/edit-and-sync-configurations-managed-by-argocd/README.md

```diff
@@ -0,0 +1,53 @@
+# edit-and-sync-configurations-managed-by-argocd
```

Comment:
I'd suggest a shorter name like `argocd-edit-and-sync` for brevity

Reply:
A shorter name is better, for sure.